src/Strategy/CommonClassesStrategy.php

```diff
@@ -76,6 +77,7 @@ final class CommonClassesStrategy implements DiscoveryStrategy
         ],
         HttpClient::class => [
             ['class' => SymfonyHttplug::class, 'condition' => [SymfonyHttplug::class, RequestFactory::class, [self::class, 'isPsr17FactoryInstalled']]],
+            ['class' => GuzzleHttp::class, 'condition' => GuzzleHttp::class],
```

Comment:
I think this is not the correct place.. this is a list with clients that implement the httplug interface..

Reply:
I think this is the place where we want guzzle: https://github.com/php-http/discovery/pull/175/files#diff-f6115264dc0232837d2b0b916ccb41b8R95